examples/search_for_values_bookmarks.py

```diff
@@ -0,0 +1,108 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (C) 2022 University of Dundee & Open Microscopy Environment.
+# All rights reserved.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+import logging
+import json
+import requests
+import sys
+import math
+
+# url to send the query
+image_value_search = "/resources/image/searchvalues/"
+# searchengine url
+base_url = "http://127.0.0.1:5577/api/v1/"
+logging.basicConfig(stream=sys.stdout, level=logging.INFO)
+
+"""
+If the user needs to search for a value whose attribute is not known e.g.
+search to find if diabetes is a part of any attribute values:
+"""
+value = "pr"
+search_url = "%S/%s%value=diabetes"
+search_url = "{base_url}{image_value_search}?value={value}".format(
+    base_url=base_url, image_value_search=image_value_search, value=value
+)
+
+# url to send the query
+image_value_search = "/resources/image/searchvalues/"
+# searchengine url
+base_url = "http://127.0.0.1:5577/api/v1/"
```

Comment:
this re-assigns `base_url` which means that editing it above has no effect

Reply:
I have modified that in the last commit

```
# search engine url
base_url = "http://127.0.0.1:5577/api/v1/"
submit_query_url = f"{base_url}resources/submitquery/"  # noqa
```

Reply:
I think that last commit didn't edit `search_for_values_bookmarks.py`.

Actually, having run through all the examples, this would be much easier if they all imported `base_url` from `utils` so you only have to edit it in one place.

Reply:
Sorry, I have fixed re-assigning `base_url `in `search_for_values_bookmarks.py`.

Yes, I think it is a good idea, now `base_url` is imported from `utils`!